spec/order_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'order'
+
+describe Order do
+  let(:order) { Order.new }
+  line = ["Cake", 2, 4.5]
+
+  it 'adds a line' do
+    expect{order.add(line)}.to change{order.lines.count}.by(1)
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.